search-app/src/cmr/search/services/parameters/converters/spatial.clj

```diff
@@ -37,3 +38,23 @@
 (defmethod p/parameter->condition :line
   [_context concept-type param value options]
   (url-value->spatial-conditions :line value))
+
+(defmethod p/parameter->condition :wkt
```

Comment:
The regex in here looks really complex, and might be prone to breaking with small differences in the input (maybe extra spaces, etc). 

1. We'd want to make sure that the parameter validation catches any errors and returns them to the user as a 400 type error before this code is called. Right now the validation might not be enough to catch all issues:
```
if-let [match (re-matches wkt-regex s)]
;; else throw error
```
2. It is probably worth adding comments here with an example string to explain what the logic is doing.
3. We should add unit tests for this code that tests a bunch of different valid inputs.